scripts/deps/update-package-json.js

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env node
+
+/*
+ * Copyright (c) 2022 - Thoughtworks Inc. All rights reserved.
```

Comment:
Can we drop this copyright?

Reply:
Sure, I copied this from my last project and overlooked this.